app/src/main/java/com/animal_villa/GameFramework.kt

```diff
@@ -6,12 +6,13 @@ import androidx.activity.ComponentActivity
 import com.example.main_animal_villa.R
 //import java.lang.reflect.Modifier
 
-class GameFramework : ComponentActivity() {
+public class GameFramework : ComponentActivity() {
```

Comment:
Specified appropriate visibility modifiers by looking at class diagram and logic https://kotlinlang.org/docs/visibility-modifiers.html